code/modules/mob/living/simple_animal/hostile/abilities_common.dm

```diff
@@ -390,3 +420,64 @@
 
 /obj/effect/proc_holder/mob_common/on_lose(mob/living/user)
 	..()
+
+/*
+ * Tame the mobs! (beastfriend)
+ */
+/obj/effect/proc_holder/mob_common/taming_mobs
+	name = "Tame"
+	desc = "Try to make hostile mobs docile."
+	action_icon = 'icons/effects/crayondecal.dmi'
+	action_icon_state = "peace"
+	banned_from_lowpop = FALSE
+	immune_to_lowpop = TRUE
+	COOLDOWN_DECLARE(taming_cooldown)
+
+/obj/effect/proc_holder/mob_common/taming_mobs/rat
+	name = "Tame - Rats"
+	desc = "Try to make rats and mice docile."
+	allowed_mobs = RTS_RATS_ALLOWED
+
+/obj/effect/proc_holder/mob_common/taming_mobs/small_critter
+	name = "Tame - Small Critters"
+	desc = "Try to make small critters docile."
+	allowed_mobs = BSF_SMALLCRITTER_ALLOWED
+
+/obj/effect/proc_holder/mob_common/taming_mobs/is_available(mob/living/user)
+	if(!..())
+		return FALSE
+	if(COOLDOWN_TIMELEFT(src, taming_cooldown))
+		user.show_message(span_alert("You can't do this for another <u>[(taming_cooldown-world.time)*0.1] seconds</u>."))
+		return FALSE
+	return TRUE
+
+// Alert animation behavior here is reversed: instead of the tamed, it is done by the ones who aren't (minus dead or already tamed).
+/obj/effect/proc_holder/mob_common/taming_mobs/Trigger(mob/user)
+	if(!..())
+		return
+	var/list/who_to_check = list()
+	var/allmobs = FALSE
+	if(!LAZYLEN(allowed_mobs))
+		allmobs = TRUE
+	else
+		who_to_check = allowed_mobs
+	for(var/mob/living/simple_animal/hostile/M in orange(6, get_turf(user)))
```

Comment:
I feel like this might let players convert a *lot* of mobs really easily, I'd consider either increasing the chance of failure, or making it a targeted ability (like sending mobs), or maybe increase the cooldown.